modules/ROOT/pages/organization.adoc

```diff
@@ -95,10 +100,11 @@ image::organization-11dbd.png[]
 
 You can also view the organization ID, client ID, and client secret. These values apply to the master organization and grant permissions for all of the business groups contained within.
 
-
 == Sharing Resources in an Organization
 
-The administrator of an organization is also the administrator of any business groups within the organization. All API versions and CloudHub environments that you create from your account are associated with this organization and are accessible only to people within the organization. So if you want to share resources with a user, you have to invite the user to join your organization, and the user must create a new account under the organization you sent the invitation from. See xref:users.adoc#inviting-users[Inviting Users] for more information.
+The organization administrator is also the administrator of any business groups (sub organizations) within the organization. All API versions and CloudHub environments that you create from your account are associated with this organization and are accessible only to people within the organization.
```

Comment:
"this organization" means the business group or the master org? Not clear what the relationship of first & second sentence is. (i.e. does 2nd stce apply to "sub org/business groups"?)